ARCHITECTURE.md

```diff
@@ -189,8 +189,8 @@ grandparent module. This restriction allows fast recalculation of the index:
 only the part corresponding to the changed files needs to be flushed.
 
 These indexes power go to class and go to symbol functionality. They are also
-used during resolve to find parent module for a file and to get the list of
-impls for a type.
+used during resolve to find the parent module for a file and to get the list of
```

Comment:
Hm, neither am I, but I'm pretty sure it must be there. It's a countable, non-abstract noun, so it can't just sit without an article. And there's only one parent module for a file, so it's `the`. I'll go find some examples to prove or rebut it :)